EventFilter/CTPPSRawToDigi/interface/CTPPSElectronicIndex.h

```diff
@@ -0,0 +1,13 @@
+#ifndef EventFilter_CTPPSRawToDigi_CTPPSElectronicIndex_H
+#define EventFilter_CTPPSRawToDigi_CTPPSElectronicIndex_H
+
+namespace ctppspixelobjects {
+  struct ElectronicIndex {
```

Comment:
Maybe I'm wrong, but I think that the comment from @slava77 was meant "Why don't you link and re-use the same identical ```struct``` in ```CondFormats/​SiPixelObjects/​interface/​ElectronicIndex.h```"?
(And if I misinterpreted the comment of Slava, I make it as my own comment...)

Reply:
I talked with the developers (who left PPS) and they told they thought it was specific for the tracker and to avoid any change in the future that could impact pps simulation, they decided to have a private version. I made the change and tested it, it works. So, I'm just waiting for the confirmation of @slava77 to push it (I could do it right away, but I'm afraid to cause unneeded growth of git history).

Reply:
no, I did not mean to use the code from CondFormats/​SiPixelObjects, I was talking about the coding pattern

Reply:
So, can I conclude I fulfilled what you meant and mark it as resolved?

Reply:
We discussed this with @slava77 : he agrees that you fulfilled his request. And ok, I now also agree that it is better not to make your code dependent on CondFormats/​SiPixelObjects, thus ok to your re-definition here of the struct. (Maybe one could make both ​SiPixelObjects and CTPPS derive from a common area for this definition... but let it for a possible further development, if someone really want to do it).
However, we still think that since the name of the struct is now ElectronicIndex, also the filename should reflect it. With its dedicated directory path, and the namespace, there is no possibility of confusion. Please then rename this file as "ElectronicIndex.h"

Reply:
done.